### ISSUE TITLE
Codelens is not keyboard actionable. 

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#126898]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:#A11y_VSCode;#A11ySev3;#A11yTCS;#A11yWCAG;#DesktopApp;#Visual Studio Code Client;#Keyboard;#WCAG2.1.1;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;

### Environment details:
Application Name: Azure Stream Analytics Tools for Visual Studio Code
Windows Version: Win10 
Visual studio code  version : -1.55.1(User Setup)

### Repro Steps:

1. Launch Visual Studio code application with installed Azure Stream Analytics Tools extension . 
2. Tab to "Explorer" section in activity bar. 
3. Press control +shift +P and edit bar will appear. 
4. Type "ASA:Create new project" and press enter to create project. 
5. New project will get created. 
6. Tab to Input item in Project Tree Section 
7. Right click on input and navigate to "ASA : Add Input" using arrow key and press enter. 
8. Select IoT hub from available option using enter key. 
9. Select "select from azure subscriptions" from available option using enter key. 
10. Type any name for input file and press enter to create Input json file. 
11. Tab to "Select Subscription" button and observe that this control is not keyboard actionable.

### Actual Result:
"Select subscription" button available in input json file is not keyboard actionable.

### Expected Result:
The keyboard focus should go on "Select subscription" button available in input json file and this button should be keyboard actionable.

### User Impact:
The Keyboard users will not able able to access the "Select Subscription" button.

### Attachment:
[A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_ConfigureSettings (1).zip](https://github.com/microsoft/vscode-docs/files/6693719/A11y_AzureStreamAnalyticsToolsForVisualStudioCode_Input_ConfigureSettings.1.zip)

**Comment by jrieken** [comment]
There is F1 > Show Codelens Command for current line
